auth-api/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>stargate</artifactId>
-    <version>1.0.3-SNAPSHOT</version>
+    <version>1.0.3</version>
```

Comment:
I'm not sure this is correct for `master` :thinking: @EricBorczuk WDYT?

Reply:
Right, should all be `1.0.4-SNAPSHOT`

Reply:
Oh weird - did something automatically change this? I don't remember messing with these, but perhaps I made an error